src/utils.js

```diff
@@ -30,11 +30,26 @@ export function windowOpen(url, { name, height = 400, width = 550 }) {
     chrome: 'yes',
   };
 
-  return window.open(
+  const shareDialog = window.open(
     url,
     platform.name === 'IE' && parseInt(platform.version, 10) < 10 ? '' : name,
     Object.keys(config).map(key => `${key}=${config[key]}`).join(', ')
   );
+
+  const interval = window.setInterval(() => {
```

Comment:
This function doesn't work `Unfortunately the popup window does not have any close event that you can listen to but there is a closed property that is true when window gets closed.` from https://stackoverflow.com/questions/15769514/window-onclose-function